### DISCUSSION TITLE
KAFKA-13391: don't fsync directory on Windows OS

### DISCUSSION THREAD

**Discussion by ostfale** [spring-projects/spring-kafka#2027] [General]
Hi, there is a problem embeddedKafka for testing according to the bug above. It seems to be fixed in Kafka. When will it be part of spring-boot-kafka?

**Comment by artembilan** [comment]
First of all there is no project like `spring-boot-kafka`.
There is Spring for Apache Kafka: https://spring.io/projects/spring-kafka with well-know Spring Framework abstraction and dependency injection features over standard Apache Kafka Client library.
And there is Spring Boot: https://spring.io/projects/spring-boot with an opinionated auto-configuration for many popular technologies including Apache Kafka over Spring for Apache Kafka.

The problem you mention happens only when we start Apache Kafka Broker in the embedded mode, so only in tests. Not so critical for production. Consider to use an external Kafka broker for your testing. For example Docker image is good one.

We will release a new version of Spring for Apache Kafka when Apache Kafka `3.0.1` is released. Not sure what else would you expect from us.
This is really more question to Apache Kafka project by itself.

**Reply by ostfale** [reply]
Thx for the information.
